pom.xml

```diff
@@ -96,14 +96,31 @@
         <type>test-jar</type>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.junit</groupId>
+        <artifactId>junit-bom</artifactId>
+        <!-- define this in properties -->
```

Comment:
The comment can be removed. It was laziness when I posted the example to not also define a properties section.

Reply:
and it was further lazyness to copy and paste it when I updated :)